pyproject.toml

```diff
@@ -43,19 +43,19 @@ requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 [project.optional-dependencies]
-# Entrypoint-specific dependencies
 fa = ["flash-attn>=2.8.0"]
 
 [tool.uv]
 no-build-isolation-package = ["flash-attn"]
+prerelease = "allow"
```

Comment:
do we need prerelease if we end up using 2.8.1 ? Or is this version a pre-release version ?